app/views/casa_org/edit.html.erb

```diff
@@ -34,7 +34,8 @@
         </div>
          <div class="col-sm align-middle my-auto">
           <% if current_organization.court_report_template.attached? %>
-            <%= link_to 'Download Current Template', rails_blob_path(current_organization.court_report_template, only_path: true), class: "btn btn-info" %>
+          <% ActiveStorage::Current.url_options = { host: request.base_url } %>
+           <%= link_to 'Download Current Template', current_organization.court_report_template.url(only_path: true), class: "btn btn-info" %>
```

Comment:
Had to remove rails_blob_path and update with url code to handle ActionView::Template::Error: Cannot get a signed_id for a new record error.

Referred this open issue - https://stackoverflow.com/questions/77201333/cannot-get-a-signed-id-for-a-new-record-with-url-for-after-attaching-pictur